src/metadataGeneration/typeResolver.ts

```diff
@@ -425,6 +425,12 @@ export class TypeResolver {
       let referenceType: Tsoa.ReferenceType;
       if (ts.isTypeAliasDeclaration(declaration)) {
         referenceType = this.getTypeAliasReference(declaration, name, node);
+      } else if (ts.isEnumMember(declaration)) {
+        referenceType = {
+          dataType: 'refEnum',
+          refName: this.getRefTypeName(name),
+          enums: [(declaration.initializer as ts.StringLiteral).text],
```

Comment:
the declaration may not have an initializer, we when resolving an entire enum, we currently do somthing like this:

```
this.current.typeChecker.getConstantValue(declaration)
```

Reply:
right